defaults/main.yml

```diff
@@ -33,7 +33,7 @@ nginx_client_body_timeout: "10"
 nginx_client_header_timeout: "10"
 nginx_send_timeout: "10"
 nginx_buffers: "on"
-nginx_client_body_buffer_size: "8k"
```

Comment:
Just general client body buffer size recommendations for nginx. I wanted to lower chances of buffer overflows and I just took the specific size from [here](http://www.cyberciti.biz/tips/linux-unix-bsd-nginx-webserver-security.html)